apps/etterlatte-saksbehandling-ui/client/src/components/person/journalfoeringsoppgave/journalpost/modal/LagreJournalpostModal.tsx

```diff
@@ -0,0 +1,113 @@
+import React, { useState } from 'react'
+import { useApiCall } from '~shared/hooks/useApiCall'
+import { Alert, Button, Heading, Modal } from '@navikt/ds-react'
+import { Journalpost } from '~shared/types/Journalpost'
+import { oppdaterJournalpost } from '~shared/api/dokument'
+import { FlexRow } from '~shared/styled'
+import { ApiErrorAlert } from '~ErrorBoundary'
+
+import { isFailure, isPending, isSuccess } from '~shared/api/apiUtils'
+import { temaTilhoererGjenny } from '~components/person/journalfoeringsoppgave/journalpost/validering'
+import { ferdigstillOppgave } from '~shared/api/oppgaver'
+import { useNavigate } from 'react-router-dom'
+import { Toast } from '~shared/alerts/Toast'
+
+interface ModalProps {
+  journalpost: Journalpost
+  oppgaveId: string
+}
+
+export default function FerdigstillJournalpostModal({ journalpost, oppgaveId }: ModalProps) {
+  const navigate = useNavigate()
+
+  const [open, setOpen] = useState(false)
+
+  const [oppdaterStatus, apiOppdaterJournalpost] = useApiCall(oppdaterJournalpost)
+  const [ferdigstillOppgaveStatus, apiFerdigstillOppgave] = useApiCall(ferdigstillOppgave)
+
+  const harGjennyTema = temaTilhoererGjenny(journalpost)
```

Comment:
Er det behov for en egen const for denne funksjonen når den kun brukes 1 gang?

Reply:
Ble brukt flere steder før, men ryddet opp og glemte å fjerne den. I fix! 👍